tea.go

```diff
@@ -577,6 +582,16 @@ func (p *Program) Quit() {
 	p.Send(Quit())
 }
 
+// Kill stops the program immediately and restores the former terminal state.
+// This final render than you would normally see when quitting will be skipped.
```

Comment:
Typo in comment here: "This final render than you would normally see when quitting will be skipped."
Should probably read: "The final render that you would normally see when quitting will be skipped." ?